README.md

```diff
@@ -36,7 +36,7 @@ See the different test jobs here:
 
 The directory structure in this openjdk-tests repository is meant to reflect the different types of test we run (and pull from lots of other locations).  The diagram below shows the test make target for each of the types, along with in-plan, upcoming additions (denoted by dotted line grey boxes).
 
-![Overview of Tests](doc/overviewOfAdoptTests.svg)
+![Overview of Tests](doc/TDoverviewOfAdoptTests.svg)
```

Comment:
Since we are using `TDoverviewOfAdoptTests.svg`, do we still need `overviewOfAdoptTests.svg` below?

Reply:
I had not decided which of the 2 styles I prefer, do you have a preference? TD/topdown or LR/leftright?

Reply:
I think TD looks prettier. But the user needs to zoom quite a bit due to the small font. 

It is better to separate it into multiple graphs? One graph shows all supported test components/types. The others show detail about each test component/type - i.e., external, perf, etc.